src/components/journal-candidate-info/journal-candidate-info.scss

```diff
@@ -0,0 +1,31 @@
+journal-candidate-info {
+  & > div {
+    height: 100%;
+    padding: 0.3em 0.75em;
+    background-color: map-get($colors, gds-grey-4);
+    border-radius: 13px;
+    -webkit-box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.55);
```

Comment:
could create a sass `mixin` for this instead. Something like this perhaps?
```
// Mixin implementation
@mixin box-shadow($top, $left, $blur, $color) {
  -webkit-box-shadow: $top $left $blur $color;
  -moz-box-shadow: $top $left $blur $color;
  box-shadow: $top $left $blur $color;
}
// Usage..
@include box-shadow(0px, 2px, 4px, rgba(0, 0, 0, 0.55));
```

Reply:
Done